ImageNet/train/google_resnet50_tpu_tensorflow18.json

```diff
@@ -0,0 +1,29 @@
+{
+  "version": "v1.0",
+  "author": "Google Cloud TPU",
+  "authorEmail": "cloud-tpu-dawnbench@googlegroups.com",
+  "framework": "TensorFlow v1.8rc1",
+  "codeURL": "https://github.com/tensorflow/tpu/blob/9de6656a779e73ac61995bd87044af21b3f37951/models/experimental/resnet_bfloat16/resnet_benchmark.py",
+  "model": "ResNet50",
+  "hardware": "GCP n1-standard-2, Cloud TPU",
+  "costPerHour": 6.595,
+  "timestamp": "2018-04-19",
+  "misc": {
+    "optimizer": "SGD with Momentum",
+    "momentum": 0.9,
+    "weightDecay": 0.0001,
+    "batchSize": 1024,
+    "warmUp": "the learning rate starts at 0, then it increases linearly per iteration for the first 5 epochs",
+    "schedule": [
+      {"learning_rate": 0.0, "epochs": 1},
+      {"learning_rate": 0.08, "epochs": 1},
+      {"learning_rate": 0.16, "epochs": 1},
+      {"learning_rate": 0.24, "epochs": 1},
+      {"learning_rate": 0.32, "epochs": 1},
+      {"learning_rate": 0.4, "epochs": 25},
+      {"learning_rate": 0.04, "epochs": 30},
+      {"learning_rate": 0.004, "epochs": 20},
+      {"learning_rate": 0.0004, "epochs": 15}
```

Comment:
This doesn't add up to the number of epochs you have in the TSV.

Reply:
Thanks for catching. Fixed it.